assignments/Joyce/week5-hw-joyce.js

```diff
@@ -0,0 +1,41 @@
+// Question: write a function `randomGroup` which have two arguments `members` and `groupNumber`
+// returns a nested array of members assign in different group randomly, the number of members
+// in each group should evenly distributed as possible.
+
+// For example
+// let members = ['Andy', 'Ben', 'Carol', 'Denny', 'Eric']
+// radnomGroup(members, 2) //return [['Andy', 'Ben', 'Denny'], ['Carol', 'Eric']]
+
+function randomGroup(members, groupNumber){
+	let membersCopy = members.slice()
+	let groups = []
+	let grpMemberCount = Math.floor(membersCopy.length / groupNumber) //2
+	let bigGrpCount = membersCopy.length % groupNumber //2
+
+	for (let i = 0; i < groupNumber; i++) {
+		let currentGroup = []
+
+		if (i < bigGrpCount) {
+			for (let i = 0; i < grpMemberCount + 1; i++) {
```

Comment:
How about extract groupMemberCount as a variable therefore you can reuse the select member logic, like below:
```
for( i < groupNumber){
  if(i < bigGrpCount){
    groupMemberCount = grpMemberCount +1
  }else{
    groupMemberCount = grpMemberCount
  }

  for(i < groupMemberCount) {
    // select memebers into group
  }
}
```